### ISSUE TITLE
Fine tune BERT model to create 4-digit SIC classifier

### ISSUE THREAD

**Issue by georgerichardson** [nestauk/industrial-taxonomy#8]
Ultimate goal: Use company descriptions from Glass to predict CH 4-digit SIC codes

There are many different possible approaches to this, particularly in regards to taking advantage of the SIC taxonomy hierarchy for iterative classification.

To establish a benchmark, we can fine tune a pre-trained transformer for a flat classification of the 4-digit level. In this case we will use:
- distilbert-base-uncased
- softmax activation (to produce a vector of probabilities rather than binary classification)

One key issue is the imbalanced classes. This is discussed further in [link to issue].
